### PR TITLE
Adding unknown error workaround

### PR DESCRIPTION
**Pull request by sishida**
Signed-off-by: Sarah Ishida <sishida@us.ibm.com>

PR for https://github.com/eclipse/codewind/issues/436

### CODE REVIEW

**Comment by tetchel** [comment]
so we don't have the "Unknown error" message anymore, it'll say something like "Error starting Codewind" or "Error installing Codewind", so maybe we can update the header etc to say "Error installing or starting Codewind" and avoid the word "unknown".

**Comment by sishida** [comment]
@tetchel Hi, Tim, Thanks for letting me know about the updated message! I have changed the title with my latest commit. 🌮

File: docs/_documentations/troubleshooting.md [hunk 1]
```diff
@@ -75,6 +75,27 @@ When using OS authentication setups (for example, AzureAD), Docker Shared Drive
 2. Create the new account with the same username but without the prefix (for example, if your AzureAD account is `AzureAD/BobSmith`, your new local account should be `BobSmith`). Use the same password as your other account.
 3. Select your new local account and click **Change account type**. Select the dropdown menu and select **Administrator**. Share the drive again in Docker.
 
+<!--
+Action/Topic: Installing Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind/issues/436
```
Comment (tetchel) [inline docs/_documentations/troubleshooting.md]: Shouldn't we link to a user issue, not the doc issue? like https://github.com/eclipse/codewind/issues/411 ?

File: docs/_documentations/troubleshooting.md [hunk 2]
```diff
@@ -75,6 +75,27 @@ When using OS authentication setups (for example, AzureAD), Docker Shared Drive
...
+0.6.0: Still present
+-->
+## Error appears after installing or updating Codewind
+After you install or update Codewind, an error might appear when you try to start Codewind.
+
+**Workaround:** Stop Codewind. Understand that the `docker system prune -a` command removes the following items:
```
Comment (tetchel) [inline docs/_documentations/troubleshooting.md]: > Stop Codewind. 
Remove the Codewind docker images. In VS Code, use the Remove Codewind Images command. In Eclipse, use the Uninstall action.
If this fails for any reason, the images can also be removed through the Docker command line using `docker image ls` to identify the Codewind images, followed by `docker image rm` to remove them.

Users should try above first, after stopping. If that still fails, then they can try the system prune.

File: docs/_documentations/troubleshooting.md (hunk 1 above)
Comment (sishida) [reply docs/_documentations/troubleshooting.md]: This is actually metadata and not viewable from the landing pages. Also, 436 links to 411, so I think this is OK as-is for our purposes here. :) Thank you for catching this, though!

File: docs/_documentations/troubleshooting.md (hunk 2 above)
Comment (sishida) [reply docs/_documentations/troubleshooting.md]: Great! I will add this in a new commit!

**Comment by tetchel** [comment]
Oh also, removing the Codewind Docker network can help:`docker network rm codewind_network`

**Comment by sishida** [comment]
@tetchel I just saw your latest comment after adding my latest commit. 😆I reorganized the workaround section. Where do you think is best to mention removing the Codewind Docker network?

**Comment by tetchel** [comment]
I would suggest
- stop 
- remove images
- remove network
- try and install/start again
- stop
- system prune

**Comment by tetchel** [comment]
nevermind, there is a new fix in the installer to remove the network when starting CW, so we can leave that step out

File: docs/_documentations/troubleshooting.md [hunk 3]
```diff
@@ -75,6 +75,33 @@ When using OS authentication setups (for example, AzureAD), Docker Shared Drive
...
+2. Remove the Codewind Docker images.
+   - In VS Code, use the `Remove Codewind Images` command.
+   - In Eclipse, use the `Uninstall` action.
+
+If removing the images with the IDE fails, try to remove them with the Docker command line instead:
+1. Stop Codewind.
+2. Use `docker image ls` to identify the Codewind images.
+3. Then, enter `docker image rm` to remove them.
+
+If installing and starting Codewind still fails, you can use Docker system prune. However, this command removes more than just the Codewind Docker images:
```
Comment (tetchel) [inline docs/_documentations/troubleshooting.md]: I think we can make this warning bigger and scarier

**Pull request merged**